doc/source/getting_started/faqs.rst

```diff
@@ -112,46 +113,53 @@ environment*:
 
 #. In a command window, use this code to set up and activate a local Python
    virtual environment::
-      
-    python -m venv pyenv   # Set up a local virtual environment
-    pyenv\Scripts\activate   # Activate the virtual environment on Windows
-    source pyenv/bin/activate.csh   # Activate the virtual environment on Linux (csh)
-    . pyenv/bin/activate   # Activate the virtual environment on Linux (bash)
 
+   .. code:: bash
+   .. vale off
```

Comment:
I deactivated vale here because it was complaining about venv appearing 2x in the first line `python -m venv venv`.  Strange, I would have thought that vale ignored rst code blocks.